db/migrate/20250902072941_sync_timerable_id_topic_id.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class SyncTimerableIdTopicId < ActiveRecord::Migration[8.0]
+  disable_ddl_transaction!
```

Comment:
The reason we want to disable the database transaction is to apply us to migrate values from one column to another without locking up the table for the entire duration. You can have a look at a previous example of how migration in batches are done: 

https://github.com/discourse/discourse/blob/485c9a5c2280a2fa4c1e1cd8e33754c9ccd2b317/db/migrate/20211201221028_migrate_email_to_normalized_email.rb#L5